data/myschema.txt

```diff
@@ -0,0 +1,58 @@
+NP in PP ---
+
+Bob [ printed ] a photo { of } my grandpa in the paper .
+Bob printed [ a photo ] { of } my grandpa in the garden .
```

Comment:
Change this to be:

    Bob printed { a photo of my grandpa in the paper } .
    Bob printed [ a photo of my grandpa in the garden ] .